core/windows/winmain.cpp

```diff
@@ -687,7 +687,10 @@ int CALLBACK WinMain(HINSTANCE hInstance,HINSTANCE hPrevInstance,LPSTR lpCmdLine
 		int dc_init(int argc,wchar* argv[]);
 		void dc_run();
 		void dc_term();
-		dc_init(argc,argv);
+		if(dc_init(argc,argv)!=rv_ok)
```

Comment:
Spacing: ``if (dc_init(argc,argv) != rv_ok)``

Reply:
@baka0815 wouldn't that also mean needing to change `int dc_init(int argc,wchar* argv[]);` to `int dc_init(int argc, wchar* argv[]);` which is outside the scope of the actual changes?

Reply:
@LoungeKatt I would also say that that change is out of scope for this PR, but since @pold500 changed that one line, the spacing should be taken care of.

Reply:
(i'm not pedantic about styling, so this looks okay to me. Would be best if changed)